pyani_plus/private_cli.py

```diff
@@ -438,7 +447,8 @@ def fragment_fasta(
     if not outdir.is_dir():
         sys.exit(f"ERROR: outdir {outdir} should be a directory")
     fragmented_files = method_anib.fragment_fasta_files(fasta, outdir, fragsize)
-    print(f"Fragmented {len(fragmented_files)} files")
+    if not quiet:
```

Comment:
When we get to logging, we should set a QUIET level and manage all user reports through the logger, rather than `print()` statements…

Reply:
Agreed.